### PR TITLE
External project vtk

### PR DESCRIPTION
**Pull request by bilke**
Added VTK as an external project.

This will be used instead of the custom catalyst vtk editions making
it much more easier to add new vtk libraries to the automatic build,
see #869.

Catalyst will be used in the future but just for in-situ vis configs.

### CODE REVIEW

**Comment by wenqing** [comment]
:+1:

**Comment by wenqing** [comment]
compilation is broken due to vtkIOParallelXML

**Comment by wenqing** [comment]
:fast_forward:

**Comment by wenqing** [comment]
Is there any other comment? If not, could someone please merge it.

File: scripts/travis/vtk.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -e
 if [ ! -d "$HOME/VTK-Install/include" ]; then
 	cd $HOME
-	wget http://www.opengeosys.org/images/dev/vtk-6.1.0.tar.gz
+	wget --no-check-certificate http://www.opengeosys.org/images/dev/vtk-6.1.0.tar.gz
```
Comment (TomFischer) [inline scripts/travis/vtk.sh]: In the file "ThirdPartyLibVersions.cmake" vtk version points to 6.3. Here the version 6.1.0 is downloaded. Is this intended?

**Comment by bilke** [comment]
Jenkins is green as well.

**Comment by bilke** [comment]
:+1:

**Comment by endJunction** [comment]
@bilke  How can one disable usage of system's vtk and force building vtk in ogs build directory?

**Comment by bilke** [comment]
This is not possible atm.

**Pull request merged**